package.json

```diff
@@ -36,6 +36,7 @@
     "copy-to-clipboard": "^3.3.1",
     "filepond": "^4.20.1",
     "lodash": "^4.17.20",
+    "material-ui-popup-state": "^4.0.1",
     "notistack": "^1.0.5",
```

Comment:
so we are building it with material ui 😅

Reply:
Yes 😄 I decided that integrating new solutions for dropdown/popover logic isn't necessary. By the way this package seems unused. I planned to use it but eventually I did not, thanks for spotting this 😄